.travis.yml

```diff
@@ -10,22 +10,21 @@ cache:
     - $HOME/.composer/cache/files
     - vendor
 
-php:
-  - 7.2
-  - 7.3
-  - 7.4
-  - nightly
-
 jobs:
   allow_failures:
     - php: nightly
   include:
     - php: 7.4
+    - php: 7.3
     - php: 7.2
       if: branch = master AND fork = false
       env:
         - UPDATE_RESOURCES=true
         - secure: "KdRIoeVMAjv63erXmG3EkOnE/UrIsgIJ7/S3yZwJq3y+c6jd7Xax6lNWRF6PcfKtNHJNhreIKA/skW5CkPktKIKqOrHrJVx5GsIQX3OqW1z2LF/qqDr7zxNWWrCCI13t7uy4GkPNU0xMwauk+5GCoyBrsxwjwf3/Lm3SA7xZTrE="
+    - php: nightly
+      install:
+        - composer self-update --preview
+        - composer install --ignore-platform-req=php
```

Comment:
This is required to ignore dependency incompatibilities with PHP 8.0. This should be remove once all dependencies are compatible with PHP 8.